scenemanager/src/main/java/io/pixsight/scenemanager/SceneManager.kt

```diff
@@ -240,14 +256,16 @@ object SceneManager {
                 ScenesMeta(
                     adapter ?: SceneAnimations.FADE,
                     creator.scenes,
-                    creator.listener
+                    creator.listener,
+                    creator.inflateOnDemand
                 )
             )
         )
-        if (creator.firstSceneId != -1) {
+
+        if (setup != null || creator.firstSceneId != -1) {
```

Comment:
Change the -1 for a constant or just initialize it to Scene.NONE

Reply:
Fix